pom.xml

```diff
@@ -487,6 +485,7 @@
       <version>2.0.2</version>
       <scope>test</scope>
     </dependency>
+    <!-- Change the 'test' scope to 'runtime' to enable console logging in examples -->
```

Comment:
Did you run into dependency order issue when you remove the test scope?

Reply:
No, I didn't, I ran the streaming ingest example from IDE without any problems. Are you running into issues? Could you try before I merge it, please?

Reply:
Signed off to unblock you, but I change it to runtime and run `mvn clean install -DskipTests`, it gives the error like below
```
[ERROR] The line 448 is not considered sorted, should be '    <!-- Change the 'test' scope to 'runtime' to enable console logging in examples -->'
[ERROR] The file /home/tzhang/Snowflake/snowflake-ingest-java/pom.xml is not sorted
```

Reply:
`mvn sortpom:sort` will auto-fix sorting issues

Reply:
I will make checking if the sorted is sorted a merge-gate only check in a separate PR.